source/jormungandr/jormungandr/utils.py

```diff
@@ -396,6 +396,7 @@ def get_pt_object_coord(pt_object):
         type_pb2.ADDRESS: "address",
         type_pb2.ADMINISTRATIVE_REGION: "administrative_region",
         type_pb2.POI: "poi",
+        type_pb2.ACCESS_POINT: "access_point",
```

Comment:
so, for you AccessPoint is a PtObject ? I thought it was not

Reply:
Since every object in `proximities_by_crowfly` (https://github.com/CanalTP/navitia/pull/3636/files#diff-77c5b35626862ae09c1bfed12bdeb812b49b47284ad3e313402ec8fc3a982bfcR170) is a `PtObject`( stop_point, car_park and access_point), converting `AccessPoint` to `PtObject` will make it easier for me to handle them instead of doing `if` every time. We also need to do this when we start to implement `AccessPoint` into `pt_ref`